### DISCUSSION TITLE
posts/k3s-traefik-rancher/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#210] [Comments]
# posts/k3s-traefik-rancher/

About

https://technotim.live/posts/k3s-traefik-rancher/

**Comment by marioacklin** [comment]
Hello - thanks for the tutorials they are really helpful. I have a question and I'm not sure if this is the place to ask but here it is. I'm following this tutorial and everything works until I deploy the traefik setup. The claim is in place but the install hangs with the message "Containers with incomplete stats: [volume-permissions]. I've tried to disable that image in the yaml file but it still doesn't load. Any thoughts?

**Comment by d70rr3s** [comment]
Hi, first of all I wanted to thank you for such a detailed and curated guide. That being said, I've a question: when doing the K3S reconfiguration step the option `--tls-san` is set with the "your load balancer ip" and I'm confused here since the whole purpose of MetalLB is to provide that piece on baremetal. Thanks in advance.

**Comment by winzlocom** [comment]
Hey, Tim.  I'm working with a k3s cluster on 4 raspberry pis - 1 control node and 3 agent nodes.  I already have a reverse proxy on my network hosted through pfSense, so is there an example of how to use traefik without requiring all of the SSL stuff?  I'm a very old Linux administrator (30+ years) but k3s is a very new thing that I'm trying to understand and get set up so that I can offload some simple web applications from my high-power servers.  Please reach out, I believe we are even local to each other geographically (go Gophers!).  Would love to connect with you about this and other things you've done to make your homelab successful.  Thanks!
